fake.go

```diff
@@ -351,7 +351,7 @@ func (fake *Fake) run(tx *Transaction) (*FakeTable, error) {
 				return nil, fmt.Errorf("unhandled operation %q", op.verb)
 			}
 		case *Element:
-			if len(obj.Value) == 0 {
+			if len(obj.Set) != 0 {
```

Comment:
I'd usually say:

```suggestion
			if obj.Set != "" {
```

(because you don't care about the length, you care about whether it's set/unset.)